src/main/java/core/basesyntax/Loterry.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Loterry {
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public String getRandomBall() {
```

Comment:
function should return Ball instance, not String
<img width="854" alt="image" src="https://user-images.githubusercontent.com/11480043/215260488-7c710f5a-e3b3-4b93-a2ab-29f8a6f5216a.png">